### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Esteban82**
Should be 2.0.1, right?

### CODE REVIEW

**Review by PaulWessel — APPROVED** [review]
I think 6.1.1 or later is required to read the changes we made in 2.0.0, so I think it is still true that 2.0.0 should be listed.  Maybe better to say 2.0.0-2.0.1

**Comment by seisman** [comment]
Perhaps a compatibility table (like this https://github.com/GenericMappingTools/pygmt#compatibility-with-gmtpythonnumpy-versions) is better?

**Comment by PaulWessel** [comment]
Yes that is true.  I guess 2.0.2 will require 6.1.1 for the same functions in 2.0.1 but 6.3.x (master) for the collections and named regions.  Maybe we should separate the updates/fixes and call the version that has collections/regions 2.1.0?  So RU bugs and other changes to 2.0.1 will be 2.0.2 while the new stuff means 2.1.0.

**Comment by Esteban82** [comment]
> the version that has collections/regions 2.1.0?

Yes, I think should be 2.1.0. We also would be arrange the belonging of some countries following the UN (M49).

**Comment by PaulWessel** [comment]
Did we resolve this issue and is this PR still needed? Master says 2.0.0 which I think is the entry point for 6.1.1.  So perhaps we do not need to merge this in?

**Comment by Esteban82** [comment]
Really I don't know. What should I do?

**Comment by meghanrjones** [comment]
I could push a compatibility table to this branch, or to a separate branch/PR if you would prefer to close this one.

**Comment by Esteban82** [comment]
Yes I think that the table will be best.

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ![GitHub](https://img.shields.io/github/license/GenericMappingTools/dcw-gmt)
 
 This repository contains the data and scripts that maintain and build
-the dcw-gmt package used by **GMT 5 or later**.  **Note**: DCW 2.0.0
+the dcw-gmt package used by **GMT 5 or later**.  **Note**: DCW 2.0.1
 requires GMT 6.1.1 or later.
```
Comment (meghanrjones) [inline README.md:8]:
```suggestion
the dcw-gmt package used by following GMT versions:

| DCW Version | Minimum GMT Version |
|-------------|---------------------|
| DCW 1.1.4   | 5 |
| DCW 2.0.0   | 6.1.1 |
| DCW 2.0.1   | 6.3.0 |
| DCW 2.0.2   | 6.1.1 |
```
I'm not sure if there's a specific GMT 5 version to list for DCW 1.1.4.

Comment (PaulWessel) [reply README.md:8]: I think it cannot read 2.* though, so probably OK to state 1.1.4 here.

**Pull request merged**